server/db/recs.js

```diff
@@ -9,6 +9,6 @@ const recsSchema = new mongoose.Schema({
   recs: []
 });
 
-const Recs = mongoose.model('Recommendations', recsSchema);
+const Rec = mongoose.model('Rec', recsSchema);
```

Comment:
Could we also make the model schema not be plural? Not sure if this is an option, but it was a good catch in the first place.

Reply:
I agree! These were already there. I'll change them.